etna/search/views.py

```diff
@@ -283,6 +312,48 @@ def get_meta_title(self) -> str:
             title += ' for "' + query.replace('"', "'") + '"'
         return title
 
+    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
```

Comment:
We can reuse [DataLayerMixin](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/analytics/mixins.py#L6) and add to this class - `BaseSearchView`.

For this implement method `get_gtm_content_group` to return Search - the content group